AntWars/Board/Ants/Ant.cs

```diff
@@ -29,6 +30,7 @@ public bool moveLeft()
         {
             Coordinates newCoords = Coords.clone();
             newCoords.X--;
+            UnitsGone++;
```

Comment:
``board.BoardObjects.move(this, newCoords);`` gibt ``true`` oder ``false`` zurück jenachdem ob das bewegen erfolgreich war, nur wenn es ``true`` ist sollte es hochgezählt werden.